habitat_sim/utils/datasets_download.py

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import os
+import pathlib
+import shutil
+import sys
+import traceback
+import zipfile
+
+data_sources = {}
+data_groups = {}
+
+
+def initialize_test_data_sources(data_path):
+    global data_sources
+    global data_groups
+    r"""Initializes data_sources and data_groups dicts with a variable data_path path.
+    """
+    # dict keyed by uids with download specs for various individual test assets and datasets
+    data_sources = {
```

Comment:
Probably for a different PR as this will require discussion, but we should add the ability to have this work with mp3d and Gibson, but require the user to provide a download URL